doc/man/task-sync.5.in

```diff
@@ -5,21 +5,97 @@ task-sync \- A discussion and tutorial for the various task(1) data
 synchronization capabilities.
 
 .SH INTRODUCTION
-Taskwarrior has several sync options, both external and built in.  If you wish
-to sync your data, choose one method only; mixing methods is going to lead to
-problems.  Each of the methods discussed have their own strengths.
+
+Taskwarrior can "synchronize" your tasks to a server. This has a few benefits:
+.br
+  - Makes your tasks accessible from multiple systems.
+.br
+  - Provides a backup of your tasks.
+.br
+  - Allows internal storage to be reused.
+
+.SH LOCAL SYNCHRONIZATION
+Taskwarrior can sync your tasks locally. While this setup does not make your
+tasks accessible from multiple systems, it does provide a convenient backup
+and allow internal storage to be reused. If you later decide to sync to a
+server, the transition is easy.
+
+Set 'sync.local.server_dir' to the directory where the synchronized data should
+be stored:
+
+    $ task config sync.local.server_dir /path/to/sync
+
+Then run 'task sync' to synchronize to this local "server".
+
+.SH SERVER SYNCHRONIZATION
+Configure each instance of Taskwarrior which you would like to keep in sync
+with the same settings.  You will need the following information from the
+adminisrator of the server:
+.br
+  - The server's URL ("origin", such as "https://tw.example.com")
+.br
+  - A client key ("client_key") identifying your tasks
```

Comment:
I think "client_key" might be better renamed "client_id". I don't think taskwarrior users need to know that some keys (unlike physical keys) do not need to be protected and can be safely shared publicly.

Reply:
#3124 (since this should be done by renaming it throughout the relevant C++ and Rust code)